Userland/Libraries/LibM/math.cpp

```diff
@@ -58,15 +61,36 @@ template<size_t value>
 constexpr size_t product_odd() { return value * product_odd<value - 2>(); }
 
 enum class RoundingMode {
-    ToZero,
-    Up,
-    Down,
-    ToEven
+    ToZero = FE_TOWARDZERO,
+    Up = FE_UPWARD,
+    Down = FE_DOWNWARD,
+    ToEven = FE_TONEAREST
 };
 
 template<typename T>
 union FloatExtractor;
 
+// This assumes long double is 80 bits, which should be the default with GCC
```

Comment:
This assumption only true on intel platforms. On MIPS/SPARC/PowerPC/aarch64, it might be IEEE 754 Quad Double, which is 128 bits.

Reply:
Indeed, let's wrap this assumption in `#if ARCH(I386) || ARCH(X86_64)`

Reply:
Should be done now